src/websql/models/customAccount.ts

```diff
@@ -0,0 +1,15 @@
+import IModel from './model';
+
+/**
+ * A model to store any other crypto assets like Tokens, NFTs, etc except the native currency of the network.
+ */
+export default interface CustomAccount extends IModel {
+  walletId: string;
+  coin: string;
+  /**
+   * Uniquer identifier for the token.
+   */
+  name: string;
+  balance: string;
+  price: string;
```

Comment:
Please make sure if `price` field is really needed here. Or else its just redundant data

Reply:
@TejasvCypherock ?

Reply:
Keeping a price field in custom account removes the need to use the parent coin DB and the way API is set up currently parent coin gathers its balance from custom account so it also avoids any interdependency.